test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -94,6 +94,33 @@ export function setup() {
 			const app = this.app as Application;
 			await app.workbench.quickaccess.runCommand('workbench.action.revertAndCloseActiveEditor');
 		});
+
+		describe('Notebook Toolbar Actions', async () => {
+
+			it('Add Code Cell', async function () {
+				const app = this.app as Application;
+				await app.workbench.sqlNotebook.newUntitledNotebook();
+				await app.workbench.sqlNotebook.addCell('code');
```

Comment:
The current add cell method uses keyboard shortcuts so we probably want to add another add cell method that uses the notebook toolbar. Right now 'add code cell' and 'add markdown cell' are testing the same things as 'basic text cell functionality' and 'basic code cell functionality'.

Reply:
Will be adding this in separate PR as this also deals with context menu dropdown that Chris is working on.

Reply:
You should move these tests out of this suite then since this implies that the tests here are for the toolbar

Reply:
I can skip them for now as I would like to be testing the add code cell/markdown cell via the toolbar specifically. I will update once have more context around the context menu for smoke tests.